.changeset/violet-ravens-draw.md

```diff
@@ -0,0 +1,6 @@
+---
+"miniflare": patch
```

Comment:
please note that we are only backporting `miniflare` and `wrangler`. The original PR contained a vitest-pool-workers patch as well, which I removed